dll/cpl/sysdm/lang/bg-BG.rc

```diff
@@ -348,12 +348,12 @@ BEGIN
     IDS_USERPROFILE_LOCAL "Local"
     IDS_USERPROFILE_ROAMING "Roaming"
     IDS_USERPROFILE_MANDATORY "Mandatory"
-    IDS_USERPROFILE_CONFIRM_DELETE "Are you sure you want to delete %s's profile?"
+    IDS_USERPROFILE_CONFIRM_DELETE "Сигурен ли сте че искате да изтриете този профил: %s's?"
```

Comment:
The `'s` part at the end should probably be deleted.

Reply:
's removed